apps/i18n/common/en_us.json

```diff
@@ -881,8 +881,8 @@
   "signinOrAge": "Sign in or provide your age to continue",
   "skipPuzzle": "Skip puzzle",
   "some": "Some",
-  "specialAnnouncementHeading": "The Hour of Code is coming",
-  "specialAnnouncementDescription": "Try our brand new Minecraft activity with your class for the Hour of Code (Dec 4-10)! This year, we’ve upgraded our servers so your students can play it signed in. They can save their progress and keep their final projects. Sign up your class today!",
+  "specialAnnouncementHeading": "Apply for TeacherCon 2018",
```

Comment:
I think we should probably use different key names for different special announcements.  One reason is that, in case we start using them for non-en, we don't have old translations showing for new announcements.

Reply:
Documenting our offline conversation - we learned that when we change a string it will default back to English until it is translated rather than stay as the translated old string. Also, we don't need to store old announcement text, so we can just change the strings and keep the keeps.